### PR TITLE
chore: apply radix colors in UI system

### PR DESCRIPTION
**Pull request by tobi-or-not-tobi**
We had a few places where we did not correctly leverage the colors. 

closes [HRZ-2901](https://spryker.atlassian.net/browse/HRZ-2901)

[HRZ-2901]: https://spryker.atlassian.net/browse/HRZ-2901?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/storybook/project.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "visual-regression-pr": {
       "executor": "@nrwl/workspace:run-commands",
       "options": {
-        "command": "chromatic --ci --only-changed --exit-zero-on-changes"
```
Comment (supproduction) [inline apps/storybook/project.json]: Assume we should revert it back

File: libs/platform/experience/src/color/colors/colored/mint.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { Color } from '../../color.model';
 
 export const mint: Color = {
   light: {
-    0: 'hsl(0, 0%, 100%)',
+    0: 'hsl(0, 0%, 0%)',
```
Comment (supproduction) [inline libs/platform/experience/src/color/colors/colored/mint.ts:5]: Is it correct that for dark and light mode we have the same color at `0` position?

**Review by supproduction — DISMISSED** [review]

File: libs/platform/experience/src/color/colors/colored/mint.ts (hunk 2 above)
Comment (tobi-or-not-tobi) [reply libs/platform/experience/src/color/colors/colored/mint.ts:5]: yes. The same "solid" color is used for the middle position (used to be our 300 color, how it is position 9). In both dark and light, the text in the solid color is white on most colours, but black on the bright colours.

File: apps/storybook/project.json (hunk 1 above)
Comment (tobi-or-not-tobi) [reply apps/storybook/project.json]: Thx for the heads up. Chromatic did not report the regressions, hence we found this workaround. Will revert it.

**Review by supproduction — APPROVED** [review]

**Pull request merged**